addons/udes_stock/models/stock_move_line.py

```diff
@@ -425,6 +425,24 @@ def _swap_tracked_items(
                     ], order="id"
                 )
 
+                # For the message thread to notify of the tracked product swapped out
+                old_mls = mls_need_changing | swapped_lot_on_assigned_mls
+                for picking, mls in old_mls.groupby("picking_id"): 
+                    picking_qty = quantity_swapping
+                    swap_out_title = "Tracked Products Swapped Out: <ul>"
+                    swap_out_text=""
+                    for ml in mls:
+                        picking_qty -= ml.product_uom_qty
+                        swapped_qty = min([quantity_swapping,ml.product_uom_qty])
```

Comment:
nit, space between `,` - mistook it for a dot

Reply:
this was added in during the formatting commit :slightly_smiling_face:

Reply:
Ah my bad, was looking at the commit on its own cause black formatting is always hard to see what's changed :+1: